haproxy/entrypoint.sh

```diff
@@ -1,30 +1,46 @@
 #!/bin/bash
-
-set -e
+set -euo pipefail
 
 DEFAULT="/usr/local/etc/haproxy/default_frontend.cfg"
 BACKEND="/usr/local/etc/haproxy/backend.cfg"
+PASSWD_DIR="/srv/storage/haproxy/passwd"
 
 # Update backend servers
-cp /usr/local/etc/haproxy/backend.cfg.template $BACKEND
-# shellcheck disable=SC2153
-for COUCHDB_SERVER in ${COUCHDB_SERVERS//,/ }
-do
-  server="  server $COUCHDB_SERVER $COUCHDB_SERVER:5984 check agent-check agent-inter 5s agent-addr $HEALTHCHECK_ADDR agent-port 5555"
-  if [[ -n "${DOCKER_DNS_RESOLVER:-}" ]]; then
-    server="${server} resolvers docker_resolver resolve-prefer ipv4"
-  fi
-  echo "${server}" >> $BACKEND
-done
+cp "${BACKEND}.template" "$BACKEND"
+
+# Read servers into array, trim whitespace
+readarray -td, SERVERS < <(echo -n "${COUCHDB_SERVERS}" | tr -d '[:space:]')
 
-# Place environment variables into config
-envsubst < $DEFAULT
-envsubst < $BACKEND
```

Comment:
I am having trouble understanding what this was even for....  😬  Just wanted to double-check, though, that we do not need it now?